app/setup/components.js

```diff
@@ -7,6 +7,9 @@ import eInput from '@/components/e-input';
 import eCheckbox from '@/components/e-checkbox';
 import eLabel from '@/components/e-label';
 import eProgress from '@/components/e-progress';
+import eInfoLabel from '@/components/e-info-label';
+import cModal from '@/components/c-modal';
+import cModalHeaderDefault from '@/components/c-modal-header-default';
```

Comment:
- [ ] This is already imported by `c-modal` should not be needed globally therefore.

Reply:
@patric-eberle Without this import it's not working.